### PR TITLE
Layout detail page

### PR DESCRIPTION
**Pull request by mbr4nt**
Opening the pull request early to begin reviewing the chages

### CODE REVIEW

File: hni-textiles-color-detail.html [hunk 1]
```diff
@@ -18,6 +18,18 @@
       --paper-menu-color: var(--asi-logo-main-type-1);
     }
     
+    p {
+      display:table-row;
+      margin:0;
+    }
+    
+    p > span {
+      border-bottom:1px solid var(--asi-logo-main-type-0);
```
Comment (mbr4nt) [inline hni-textiles-color-detail.html]: @felipebbtoledo there's not main type 0

File: hni-textiles.html [hunk 2]
```diff
@@ -3,7 +3,10 @@
 <!-- Iron elements -->
 <link rel="import" href="../iron-ajax/iron-ajax.html" />
 <link rel="import" href="../iron-pages/iron-pages.html">
+<<<<<<< HEAD
```
Comment (mbr4nt) [inline hni-textiles.html]: @felipebbtoledo don't commit versions with conflicts. You need to solve conflicts before committing.

**Comment by felipebbtoledo** [comment]
@mbr4nt Please fill in the propname fields.

File: hni-textiles-color-detail.html [hunk 3]
```diff
@@ -114,37 +170,38 @@
...
             </template>
           </div>
-
           <template is="dom-repeat" items="{{_expandDictionary(response.properties)}}">
             <p>
-              <div propname>{{item.key}}</div>
-              <template is="dom-repeat" items="{{item.values}}">
-                <div propvalue>{{item}}</div>
-              </template>
+              <span propname>{{item.key}}:</span>
```
Comment (mbr4nt) [inline hni-textiles-color-detail.html]: @felipebbtoledo can't have the ":" here. this will break it.

Comment (felipebbtoledo) [inline hni-textiles-color-detail.html]: @mbr4nt seriously?! Ok then lol

Comment (mbr4nt) [inline hni-textiles-color-detail.html]: @felipebbtoledo  Yep, it needs to be just the binding expression in there, nothing else. If you really need the ":" we can either:

- make a that concats it, then bind to the function
- make a new span for just the <span>:</span>

Comment (felipebbtoledo) [inline hni-textiles-color-detail.html]: @mbr4nt I will remove it for now.

**Comment by felipebbtoledo** [comment]
@mbr4nt I need your help with the visualization section. The download button and change product dropdown list must be within the asi-sku-gallery to make possible the absolute positioning at the bottom of the image. This way it will be in the correct position in all devices. The way it is now, inside the visualization-container div, it won't work because the thumbnails position and size will vary depending on the device.

**Comment by felipebbtoledo** [comment]
@mbr4nt Also, we need to move the label "Change product" inside the dropdown list. Currently, the dropdownlist shows the current product selected.

**Comment by felipebbtoledo** [comment]
@mbr4nt And how can I make the visualization section to be 75% width as the big image at the top of the page so they align? I tried to use the pre existing classes like md3,4,6,8 and lg3,4,6,8 and there is none that achieve that.

File: hni-textiles.html (hunk 2 above)
Comment (mbr4nt) [inline hni-textiles.html]: @felipebbtoledo will you please fix this now?

Comment (felipebbtoledo) [inline hni-textiles.html]: @mbr4nt done.
